src/vs/workbench/contrib/debug/browser/media/debugHover.css

```diff
@@ -32,6 +32,17 @@
 	border-bottom: 1px solid rgba(128, 128, 128, 0.35);
 }
 
+.tip {
+	padding-left: 15px;
+	padding-right: 2px;
+	font-size: 11px;
+	line-height: 18px;
+	word-break: normal;
+	height: 18px;
+	white-space: pre;
+	border-top: 1px solid rgba(128, 128, 128, 0.35);
```

Comment:
We usualy try not to introduce colors directly into `css` so that they can be customised.